exist-ant/src/main/java/org/exist/ant/XMLDBExtractTask.java

```diff
@@ -273,6 +260,18 @@ private void writeXMLResource(final XMLResource res, final File dest) throws IOE
         }
     }
 
+    private Writer getWriter(XMLResource res, File dest) throws XMLDBException, FileNotFoundException {
+        final Writer writer;
+        if (dest.isDirectory()) {
+            String fname = res.getId();
+            final File file = new File(dest, fname);
+            writer = new OutputStreamWriter(new FileOutputStream(file), UTF_8);
```

Comment:
No biggie, but for Java 8 this could be:

```java
writer = Files.newBufferedWriter(file, UTF_8);
```

Reply:
That is a nice one, I will follow-up on that. Not sure what to use for the PathOptions yet...